src/v2/layouts/Button/Button.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export interface ButtonProps {
```

Comment:
We also need props like:
```suggestion
export interface ButtonProps {
  text,
  onClick,
  type: 'submit' | 'button'
}
```

Reply:
Resolved